0.27.1 failed to release

Release tries to make a new release:
https://github.com/nektos/gh-act/actions/runs/12566374661/job/35031701824
> android/amd64 requires external (cgo) linking, but cgo is not enabled
> Error: Process completed with exit code 1.

The cause is:
- https://github.com/cli/cli/issues/10084

There's no traction on it.

Unfortunately this means that anyone using `gh act` gets a complaint that `act` isn't current and there's nothing they can do about it...

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -35,6 +35,6 @@ jobs:
           key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
           restore-keys: |
             ${{ runner.os }}-go-
-      - uses: cli/gh-extension-precompile@v1
+      - uses: cli/gh-extension-precompile@v2
         with:
           go_version: ${{ env.GO_VERSION }}

